src/shapes.ts

```diff
@@ -218,6 +218,7 @@ export interface WindowOptions {
     title?: string;
     toShowOnRun?: boolean;
     transparent?: boolean;
+    _type?: 'OF_error_box:app_initialization'|'OF_error_box:main_process'|'OF_error_box:renderer_crash';
```

Comment:
Great tip, it should work now, but hopefully in the future if we are grabbing stuff from this shapes file it won't create circular dependency fails

Reply:
Just learnt you can't do it **nicely**, because we'd have to declare a type for each const to be able to use it in that interface. So in either case we'd have to hard code the string 2 times.

![image](https://user-images.githubusercontent.com/5790216/43590517-2c3e88d4-963f-11e8-8972-d9459f9008c0.png)

https://stackoverflow.com/questions/44080165/cannot-find-name-defined-in-same-file-how-to-achieve-union-type-of-const-values

Reply:
String enum?

Reply:
@pbaize brilliant!